docs/docs-beta/docs/guides/monitor-alert/custom-logging.md

```diff
@@ -3,4 +3,96 @@ title: "Setting up custom logging"
 sidebar_position: 20
 ---
 
-# Setting up custom logging
+
+# Custom loggers
```

Comment:
Are custom loggers also what is to be used for integration with Datadog or Sentry?

I can imaging users landing here looking for that.

Reply:
no. that's basically formatting console log output, not about sending it elsewhere.

Reply:
ok thank you for clarifying